src/content/reference/gsctl/show-cluster.md

```diff
@@ -64,7 +63,7 @@ The output lines in detail:
 - **Worker VM size:** (_only on Azure_) VM size used for worker nodes
 - **CPU cores in workers:** total number of CPU cores in all worker nodes in this cluster
 - **RAM in worker nodes (GB):** total amount of memory in all worker nodes in this cluster
-- **Storage in worker nodes (GB):** total amount of local storage in all worker nodes in this cluster
+- **Storage in worker nodes (GB):** (only for KVM) total amount of local storage in all worker nodes in this cluster
```

Comment:
Above we have `**Worker VM size:** (_only on Azure_)` (not undrescores). I think would be good to take consistent approach. I personally don't case which one.

Reply:
Thanks, changed!